tests/generators/test_classic.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+import xgi
+from xgi.exception import XGIError
```

Comment:
Nitpick: this import also seems unnecessary.

Reply:
yep! fixed it.